### PR TITLE
Map Feature

### PR DESCRIPTION
**Pull request by EPP100**
A map view to display where photos were taken

### CODE REVIEW

File: web/src/lib/components/map-page/photos-map-view.svelte [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+<div class="map">
+	<div class="h-full w-full" id="map" />
+</div>
+
+<link
+	rel="stylesheet"
+	href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+	integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+	crossorigin=""
+/>
```
Comment (bo0tzz) [inline web/src/lib/components/map-page/photos-map-view.svelte:157]: Please install this as an NPM dependency instead of using a CDN.

**Review by bo0tzz — CHANGES_REQUESTED** [review]
Thanks for the PR! For now, just a few high level thoughts from me.

File: server/immich-openapi-specs.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
             "name": "assetId",
             "required": false,
             "in": "query",
-            "description": "Only returns albums that contain the asset\nIgnores the shared parameter\nundefined: get all albums",
+            "description": "Only returns albums that contain the asset\rIgnores the shared parameter\rundefined: get all albums",
```
[the remark this answers is not in the document]
Comment (colin715) [reply server/immich-openapi-specs.json:20]: FYI, I believe that EPP said on discord that he wouldn't be able to work on this further

**Comment by bo0tzz** [comment]
Superseded by #2355

**Comment by jrasm91** [comment]
Thanks for kick starting this!

**Pull request closed without merging**